src/pages/content/word-list.mdx

```diff
@@ -1,12 +1,707 @@
+
+Use clear or cancel instead of deselect. As in, “Clear these checkboxes” and
+“Cancel this selection.”
+
+### done
+
+Avoid done since it doesn’t provide enough information. Button labels should
+make it clear what’s going to happen when the user clicks it.
+
+<BestPractice>
+  <Do>
+    <p>
+      Description: Settings updated <br />
+      Button label: Go to Admin Center
+    </p>
+  </Do>
+  <Dont>
+    <p>
+      Description: Settings updated
+      <br />
+      Button label: Done
+    </p>
+  </Dont>
+</BestPractice>
+
+### drop-down
+
+This noun describes a UI element that lets users pick from a series of options.
+Always hyphenate it.
+
+### for example, e.g.
+
+Keep it conversational by using for example. The scholarly way to say “for
+example” is by using “e.g.,” which stands for exempli gratia.
+
+### either
+
+Use either only for a decision between two choices.
+
+<BestPractice>
+  <Do>
+    <p>You may choose either bacon or tofu.</p>
+  </Do>
+  <Dont>
+    <p>You may choose either bacon, snake meat, or tofu.</p>
+  </Dont>
+</BestPractice>
+
+### email, emails
+
+No hyphen needed.
+
+### feedback
+
+Use give feedback or for some Early Access Program (EAP) links, share your
+thoughts.
+
+<BestPractice>
+  <Do>
+    <p>Give feedback</p>
+  </Do>
+  <Dont>
+    <p>Provide feedback</p>
+  </Dont>
+</BestPractice>
+
+### full and fully
+
+Avoid full and fully unless you’re writing marketing copy. Even then, only use
+to mean complete.
+
+<BestPractice>
+  <Do>
+    <ul>
+      <li>Full-featured agent workflow</li>
+      <li>Full ticket history</li>
+    </ul>
+  </Do>
+  <Dont>
+    <ul>
+      <li>Fully customizable</li>
+      <li>Full live chat software functionality</li>
+    </ul>
+  </Dont>
+</BestPractice>
+
+### greater, better
+
+Avoid using subjective terms when talking about system requirements or versions.
+Use objective terms instead.
+
+<BestPractice>
+  <Do>
+    <ul>
+      <li>Update to the latest version of Google Chrome</li>
+      <li>Chat requires Microsoft Edge v.14 or higher</li>
+    </ul>
+  </Do>
+  <Dont>
+    <ul>
+      <li>Update to the best version of Explore</li>
+      <li>Chat requires Microsoft Edge v.14 or better</li>
+    </ul>
+  </Dont>
+</BestPractice>
+
+### Help Center
+
+Capitalize both words. Also, refer to it as Help Center, not the Help Center.
+
+Example: "Edit articles in Help Center."
+
+A possessive pronoun is okay.
+
+Example: “Edit articles in your Help Center.”
+
+### help desk
+
+Zendesk makes customer service software, not help desk software. The term help
+desk should only be used when specifically addressing IT or help desk issues.
+
+In most cases, you can use Zendesk Support instead.
+
+Example: "You can use Zendesk Support to improve your relationship with your
+customers."
+
+### hint
+
+Don’t use it as a heading for a type of note. Use tip instead.
+
+<!-- markdownlint-disable -->
+
+### i.e.
+
+Abbreviation for id est, meaning “that is.” Use that is or in other words
```

Comment:
Abbreviation for id est, meaning “that is.” Use _that is_ or _in other words_
instead.